qa/suites/rgw/cloud-transition/tasks/cloud_transition_s3tests.yaml

```diff
@@ -0,0 +1,39 @@
+tasks:
+- install:
+- ceph:
+- rgw:
+   client.0:
+     port: 8000
+   client.1:
+     port: 8001
+- s3tests:
+    client.0:
+      force-repo: https://github.com/soumyakoduri/s3-tests.git
+      git_remote: https://github.com/soumyakoduri/
```

Comment:
This will be updated once https://github.com/ceph/s3-tests/pull/442 gets merged.